diff -Nurp a/common/mp4v2/mpeg4ip.h b/common/mp4v2/mpeg4ip.h
--- a/common/mp4v2/mpeg4ip.h	2013-02-27 12:36:55.108660504 +0100
+++ b/common/mp4v2/mpeg4ip.h	2013-02-27 12:37:47.915936284 +0100
@@ -120,14 +120,6 @@
 #endif
 #include <sys/param.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-char *strcasestr(const char *haystack, const char *needle);
-#ifdef __cplusplus
-}
-#endif
-
 #define OPEN_RDWR O_RDWR
 #define OPEN_CREAT O_CREAT 
 #define OPEN_RDONLY O_RDONLY
