diff -Nurp a/fbida-2.09/GNUmakefile b/fbida-2.09/GNUmakefile
--- a/fbida-2.09/GNUmakefile	2013-09-20 11:30:41.586371250 +0200
+++ b/fbida-2.09/GNUmakefile	2013-09-20 12:09:55.426446573 +0200
@@ -31,7 +31,7 @@ include $(srcdir)/mk/Autoconf.mk
 ac_jpeg_ver = $(shell \
 	$(call ac_init,for libjpeg version);\
 	$(call ac_s_cmd,echo -e '\#include <jpeglib.h>\nJPEG_LIB_VERSION' \
-		| cpp | tail -n 1);\
+		| cpp $(CFLAGS) | tail -n 1);\
 	$(call ac_fini))
 
 define make-config
